### ISSUE TITLE
[ENHANCEMENT] langchain tracing option to display the raw openai JSON messages.

### ISSUE THREAD

**Issue by taoari** [Arize-ai/phoenix#4530]
**Is your feature request related to a problem? Please describe.**

When I enabled tracing for Langchain's ChatOpenAI, the input and output were shown in LangChain's custom JSON format (see screenshot). However, this format is cluttered with unnecessary details, making it hard to follow. What I need is access to the original OpenAI API calls, which are much cleaner and more familiar to developers. Since OpenAI tracing is already supported in Phoenix, having the option to view the raw OpenAI JSON messages would significantly simplify debugging. Moreover, the OpenAI API has become a standard that other frameworks, like vLLM and HuggingFace’s text generation interface, are adopting.

<img width="1344" alt="Screenshot 2024-09-05 at 2 49 53 PM" src="https://github.com/user-attachments/assets/f3fb2bed-e8fe-429c-b022-4e09904d24b5">

**Describe the solution you'd like**

This screenshot below is for OpenAI tracing, it's JSON format is much cleaner (simply messages of "role" and "content"). I would expect when I click on ChatOpenAI, it will show cleaner messages like this.

<img width="1344" alt="Screenshot 2024-09-05 at 3 12 55 PM" src="https://github.com/user-attachments/assets/c45e523d-2622-429e-bf19-0ae653ab21c3">

**Describe alternatives you've considered**

**Additional context**

**Comment by mikeldking** [comment]
thanks for the feedback @taoari - we will have some unification of attributes to API calls in the future. Will keep you updated.

You could always instrument the actual OpenAI network call if needed
